index.js

```diff
@@ -2,16 +2,19 @@ var tagFilter = require("./lib/tag_filter");
 var groupFilter = require("./lib/group_filter");
 var singleTestFilter = require("./lib/single_test_filter");
 var fileFilter = require("./lib/file_filter");
+var skipTagFilter = require('./lib/skip_tag_filter');
 
 var plugin = {
   initialize: function (argv, options) {
     plugin.settings.initialize(argv, options);
   },
   iterator: require("./lib/get_tests"),
   filters: {
+    group: groupFilter,
```

Comment:
These filters are run in order, so moving `group` up higher will slightly improve performance by reducing the number of tests that need to be scanned for tags when both `--group` and `--(skip)tags` are included